jupyter_book/commands/__init__.py

```diff
@@ -72,11 +82,15 @@ def build(path_book, path_output, config, toc, warningiserror, build):
         extra_extensions = config_yaml.pop("sphinx", {}).get("extra_extensions")
 
     # Builder-specific overrides
-    if build == "pdf_html":
+    if build == "pdfhtml":
         book_config["html_theme_options"] = {"single_page": True}
 
-    OUTPUT_PATH = path_output if path_output is not None else PATH_BOOK
-    OUTPUT_PATH = Path(OUTPUT_PATH).joinpath("_build/html")
+    BUILD_PATH = path_output if path_output is not None else PATH_BOOK
```

Comment:
@choldgraf have I stuffed up the merge conflict fix here. Not sure why any differences are coming up for `pdfhtml`. Should I revert to what is on master here?